aries-core/src/js/components/core/Identifier/Identifier.js

```diff
@@ -10,23 +9,25 @@ export const Identifier = ({
   subTitle,
   size,
   ...rest
-}) => (
-  <Box align="center" {...rest}>
-    {children}
-    <Box>
-      {level ? (
-        <Heading level={level} margin="none">
-          {title}
-        </Heading>
-      ) : (
-        <Text size={size} weight="bold" color="text-strong">
-          {title}
-        </Text>
-      )}
-      <Text size={size}>{subTitle}</Text>
+}) => {
+  return (
```

Comment:
Seems like we had an unnecessary format change  here. Can you run prettier format on this file to double check?

Reply:
Hmm, I'm running prettier on it and it doesn't seem to change... it's a weird format change though